_docs/reference/release-roadmap.md

```diff
@@ -32,6 +33,7 @@ Istio 0.3 will be our third release, focused on performance, scale, and stabilit
 - Client-oriented telemetry collection and distribution.
 - Global load balancing with autoscaling.
 - Support for the Open Service Broker APIs.
+- Authentication support including pluggable CA, container identity, etc.
```

Comment:
How about:

Support for pluggable Authentication providers.

Or something like that.